### PR TITLE
updated description of where examples for package push live

### PR DESCRIPTION
**Pull request by benPearce1**
[sc-69469](https://app.shortcut.com/octopusdeploy/story/69469)

The **Show Examples** button has been removed and replaced with content in the help sidebar

image

### CODE REVIEW

File: dictionary-octopus.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ALLUSERSPROFILE
 apikey
+YOURAPIKEY
```
Comment (benPearce1) [inline dictionary-octopus.txt:3]: This is the sample api key used in the page that I changed, it is used in a few other places, but there are a number of other examples of placeholder api keys used

Comment (steve-fenton-octopus) [reply dictionary-octopus.txt:3]: We're trying to standardize on the also-used `YOUR-API-KEY` which is a little more readable.

Comment (benPearce1) [reply dictionary-octopus.txt:3]: I agree that it is more readable but it suggests a different format. Api Keys are always `API-XXXXX` with no additional hyphens.

**Review by IsaacCalligeros95 — APPROVED** [review]
LGTM

**Pull request merged**